setup.py

```diff
@@ -74,5 +74,13 @@
     version = '{version}'
 """.lstrip()
 
+current_path = os.path.abspath(os.path.dirname(__file__))
+
+def read_file(*parts):
+    with open(os.path.join(current_path, *parts), encoding='utf-8') as reader:
+        return reader.read()
+
 setup(use_scm_version={'write_to': os.path.join('astrocut', 'version.py'),
-                       'write_to_template': VERSION_TEMPLATE})
+                       'write_to_template': VERSION_TEMPLATE},
+     long_description=read_file('README.rst'),
+     long_description_content_type='text/markdown')
```

Comment:
does this work? it's not markdown, but restructured text.

it should be: `text/x-rst`

from: https://packaging.python.org/en/latest/specifications/core-metadata/#description-content-type

Reply:
Ah, hmm, that didn't give me any problems, but that's probably because it hasn't gotten to the description part yet. I'll update accordingly, thanks